workers_manager.py

```diff
@@ -100,6 +110,33 @@ def _pip_install_helper(package_names):
     for package in package_names:
       pip_main(['install', '-q', package])
 
+  def _poll_wrapper(self, worker_obj, client, userdata, c):
+    if c.payload.decode('utf-8') != '':
+      return
+    poll_device_supported = 'poll_device' in worker_obj.status_update.__code__.co_varnames
```

Comment:
It could also benefit from some devices getter. When there is None / not defined / only one device  ( ? ) return from worker_obj.devices then it does not support